tests/fast/ConfigIncrement.toml

```diff
@@ -9,11 +9,17 @@ testTitle = 'ConfigIncrement'
   incrementActors = 2
   incrementsPerActor = 10
   meanSleepWithinTransactions = 0.01
-  meanSleepBetweenTransactions = 0.1
+  meanSleepBetweenTransactions = 10
```

Comment:
Is this 10 second sleep necessary? What is the result of reducing the mean sleep time?

Reply:
I made this change so that it was more likely there would be transactions occurring during a coordinator change. Probably this should be reverted to 0.1 for this specific test and a new test should be added to specifically test coordinator changes combined with transactions. With the 0.1 second sleep, my experience was that all the transactions completed very quickly and the coordinator change either took place entirely before or after the increment workload.